src/main/java/io/seqera/tower/cli/responses/ComputeEnvs/ComputeEnvExport.java

```diff
@@ -0,0 +1,35 @@
+package io.seqera.tower.cli.responses.ComputeEnvs;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.seqera.tower.JSON;
+import io.seqera.tower.cli.responses.Response;
+import io.seqera.tower.cli.utils.FilesHelper;
+import io.seqera.tower.model.CreateComputeEnvRequest;
+
+public class ComputeEnvExport extends Response {
+
+    public final CreateComputeEnvRequest request;
+    public final String fileName;
+
+    public ComputeEnvExport(CreateComputeEnvRequest request, String fileName) {
+        this.request = request;
+        this.fileName = fileName;
+    }
+
+    @Override
+    public String toString() {
+        String configOutput = "";
+
+        try {
+            configOutput = new JSON().getContext(CreateComputeEnvRequest.class).writerWithDefaultPrettyPrinter().writeValueAsString(request);
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
+        }
+
+        if (fileName != null) {
```

Comment:
> Not sure if we need to allow to export into a file on this first iteration.

I believe the export to file is important because it allows you to execute and export and import command set in a non-sequential way by using the file name as a reference. I think it is a matter of how do we wish to use the E/I commands. Nevertheless, the export command does not require a `filename` to be defined.

> I'd add the option to accept `-` as a possible `fileName` value.

Don't quite understand this, sorry.

Reply:
Using `-` as a filename to mean `stdin` or `stdout` is a convention that a lot of command line programs use. So if the user sets `-` as filename the expected behaviour is to print to `stdout` (the same that we do if no filename is given)